graphsdk/src/androidTest/java/com/microsoft/graph/serializer/ByteArraySerializerTests.java

```diff
@@ -0,0 +1,21 @@
+package com.microsoft.graph.serializer;
+
+import android.test.AndroidTestCase;
+
+/**
+ * Test cases for {@see ByteArraySerializer}
+ */
+public class ByteArraySerializerTests extends AndroidTestCase {
+
+    public void testByteArraySerializer() throws Exception {
```

Comment:
Please break this into 2 different test cases, byte array and string serialization/deserialization,